deploy/charts/cert-manager/templates/networkpolicy-egress.yaml

```diff
@@ -0,0 +1,28 @@
+{{- if .Values.webhook.networkPolicy.enabled }}
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: {{ template "webhook.fullname" . }}-allow-egress
+  namespace: {{ include "cert-manager.namespace" . }}
+spec:
+  egress:
+  - ports:
```

Comment:
In case we want to configure multiple egress rules, it would be nice to have this whole array configurable.

Reply:
What other rules you have in mind what would be still part of Cert-manager stack and would not fit into "create separate networkPolicy" for it case?

Reply:
Im just trying to think how not to overload the config so it would end-up "full NP file" in values.yaml file

Reply:
based on https://github.com/cert-manager/cert-manager/issues/2334#issuecomment-787726661, you might want to allow access to some ports only for a specific range of ips, while other ports should be accessible for all ips

Reply:
make sense  :) will update